src/components/timeline/TrackProcessCPUGraph.js

```diff
@@ -15,7 +15,7 @@ import {
   getProfileInterval,
 } from 'firefox-profiler/selectors/profile';
 import { getThreadSelectors } from 'firefox-profiler/selectors/per-thread';
-import { ORANGE_50 } from 'photon-colors';
+import { GREY_40 } from 'photon-colors';
```

Comment:
If we want to do grey, I'd suggest to use GREY_50 instead, as it's more contrasted.

Against white, GREY_50 is 4.74 which is > 4.5 so it's OK. GREY_40 is only 2.14 which isn't good enough.

Reply:
I also experimented with GREY_50 but ended up liking 40 more. But if it's a matter of accessibilty, it's obviously more important. So changed it, thanks.